verilog/transform/obfuscate.cc

```diff
@@ -141,6 +144,38 @@ static absl::Status VerifyEquivalence(absl::string_view original,
   return absl::OkStatus();
 }
 
+// Find all modules and collect interface names
+absl::Status CollectInterfaceNames(absl::string_view content,
+                                   verible::StringSet* if_names) {
+  VLOG(1) << __FUNCTION__;
+
+  const auto analyzer =
+      verilog::VerilogAnalyzer::AnalyzeAutomaticMode(content, NULL);
+  const auto lex_status = ABSL_DIE_IF_NULL(analyzer)->LexStatus();
+  const auto parse_status = analyzer->ParseStatus();
+
+  if (lex_status.ok() && parse_status.ok()) {
+    const auto& syntax_tree = analyzer->SyntaxTree();
+    const auto mod_headers =
+      FindAllModuleHeaders(*ABSL_DIE_IF_NULL(syntax_tree).get());
+
+    // For each module, collect all indentifiers under the module
+    // header tree into the StringSet.
+    for(auto const& mod_header: mod_headers) {
```

Comment:
Most other places, we've been using `const auto&` instead of `auto const&`, though I know they're equivalent.